tools/qmlformat.py

```diff
@@ -28,7 +29,16 @@ def find_qt_version():
 
 
 def main(argv=None):
-    qmlformat_executable = shutil.which("qmlformat")
+    # First look up at the most common location for QT6 which is not in PATH
+    # This applies to Debian, Arch and Fedora (i686)
+    qmlformat_executable = shutil.which(
+        "qmlformat", os.F_OK, "/usr/lib/qt6/bin"
```

Comment:
The mode argument is wrong, now it will find files that exist but aren't executable. Please just remove it and use a keyword argument for `path`:

```python
        "qmlformat", path="/usr/lib/qt6/bin"
```

Also, please create a list with possible installation paths, so that isn't not redundant when checking for `qmlformat-qt6` or `qmlformat`:

```python
unix_paths = (
    "/usr/lib/qt6/bin", 
    "/usr/lib64/qt6/bin", 
)
path = os.sep.join(unix_paths) if sys.platform != "win32" else None
```